client/app/index.ejs

```diff
@@ -90,7 +91,8 @@
             </div>
         </div>
 
-        <div id="managerPreload" class="d-flex text-center" data-ng-class="managerPreloadHide">
+        <div id="managerPreload" class="d-flex text-center"
+            data-ng-class="!!navbar && managerPreloadHide">
```

Comment:
Using the class in the variable `managerPreloadHide` only if `navbar` is defined ? Wouldn't it be clearer in a function ?

Reply:
Sadly, it's like that in every manager. In Web, the classname is added on `$stateChangeSuccess`.
Here I just check that the navbar is defined before adding the class, to avoid the possibility of a blank space.